app/controllers/shop/Shop.php

```diff
@@ -1800,6 +1951,9 @@ public function suggestions($pos = 0)
 
                 $c = uniqid(mt_rand(), true);
                 unset($row->cost, $row->details, $row->product_details, $row->barcode_symbology, $row->cf1, $row->cf2, $row->cf3, $row->cf4, $row->cf5, $row->cf6, $row->supplier1price, $row->supplier2price, $row->cfsupplier3price, $row->supplier4price, $row->supplier5price, $row->supplier1, $row->supplier2, $row->supplier3, $row->supplier4, $row->supplier5, $row->supplier1_part_no, $row->supplier2_part_no, $row->supplier3_part_no, $row->supplier4_part_no, $row->supplier5_part_no);
+                $convertedData = $this->getEnglishToArabic($row->name);
```

Comment:
seems it will always translate from english to arabic even if customer search in english as well.